### ISSUE TITLE
Textarea??

### ISSUE THREAD

**Issue by kotko** [react-native-training/react-native-elements#267]
Does your library have a textarea?

**Comment by bozzmob** [comment]
Hi, Not sure what exactly you mean by Textarea. But, if you are referring to Text component. We don't explicitly support it here, you can use the [Text](https://facebook.github.io/react-native/docs/text.html) component directly from react native and add styles that suits your UI.

I don't see any potential use of having a Text component on top of what react native already gives us. Please share your thoughts if you have anything in mind.

**Comment by kotko** [comment]
I mean a field for entering a message, for example

**Comment by binoy14** [comment]
@kotko Could you please provide the example?

**Comment by luchoster** [comment]
You can just use the {TextInput} from 'react-native' and define multineline.

```
<TextInput 
    multiline = {true}
    numberOfLines = {4}
    value='Message'
/>
```

**Comment by bozzmob** [comment]
Hi @kotko , Luchoster 's solution looks universally good. You can surely style it the way you want(or how well it would match with your app).

I would be closing this issue, as the solution looks good to me. Let me know if you need any more help.

**Comment by A** [comment]
Hello @bozzmob! 
I see this in the docs:
> THIS COMPONENT INHERITS ALL NATIVE TEXTINPUT PROPS THAT COME WITH A STANDARD REACT NATIVE TEXTINPUT ELEMENT, ALONG WITH THE FOLLOWING

But when I'm adding `multiline={true}` to `<FormInput />` I see that it become multiline but it doesn't resize. Is it bug? 

<img width="405" alt="screen shot 2017-06-28 at 20 36 59" src="https://user-images.githubusercontent.com/1410106/27651597-8daf6d92-5c41-11e7-9c0b-112dec7214b0.png">

**Comment by A** [comment]
Looks like it's a bug. I fixed it in #455
